<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SharedlibNameServerFrame</class>
 <widget class="QMainWindow" name="SharedlibNameServerFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SharedlibName Server</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="outputLayout">
      <item>
       <widget class="QLabel" name="outputLabel">
        <property name="text">
         <string>Server Display Message:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="outputEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="inputLayout">
      <item>
       <widget class="QLabel" name="inputLabel">
        <property name="text">
         <string>Server Input Here:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="inputEdit">
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">actionCreate
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionCreate"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCreate"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCreate">
   <property name="icon">
    <iconset resource="sharedlib_name_server.qrc">
     <normaloff>:/images/create.png</normaloff>
     :/images/create.png
    </iconset>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
   <property name="toolTip">
    <string>Great creativity</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="sharedlib_name_server.qrc">
     <normaloff>:/images/exit.png</normaloff>
     :/images/exit.png
    </iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit SharedlibName Server</string>
   </property>
   <property name="shortcut">
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="sharedlib_name_server.qrc">
     <normaloff>:/images/info.png</normaloff>
     :/images/info.png
    </iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About SharedlibName Server</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
